utils/format.go

```diff
@@ -1277,3 +1252,59 @@ func FormatTimestampUInt64(ts uint64) template.HTML {
 func FormatEth1AddressFull(addr common.Address) template.HTML {
 	return FormatAddress(addr.Bytes(), nil, "", false, false, true)
 }
+
+func FormatHeaderHash(address []byte) template.HTML {
```

Comment:
What is this supposed to do?
Currently it only shows the first 4 and last 4 digits of the validator pubkey in orange (primary).

Reply:
This function works in tandem with the truncateMiddle function in `layout.js`. In the Figma the public key of the validator gets an ellipsis in the middle when there is not enough space, which is currently not supported by CSS, so I had to write that functionality myself.
Did you see only the first and last 4 digits upon opening the validator page? I would assume you didn't have the right `layout.js` cached, but if it ever happens again please show me.

Reply:
Yes, it was just that, e.g. "0xabcdabcd". Like you said it was probably a cache issue.